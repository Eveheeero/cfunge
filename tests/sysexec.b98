< v ,a =  v# "echo 'Hi there (and now a ls):'; ls" 0
          > 0a"detcelfer =" >:#,_@
  > 0" :)0 eb dluohs( lavteR" >:#,_ $ . a,   v

v ,a   = "echo 'Will run: exit 2'; exit 2" 0 <
> 0" :)2 eb dluohs( lavteR" >:#,_ $ . a,@

This program should ouput (on a POSIX system):
	Hi there (and now a ls):
	<list of files in current directory, as ls outputs it>

	Retval (should be 0): 0
	Will run: exit 2

	Retval (should be 2): 2
There should be no indention in the output. (It was done here to it clear this
line is not part of the output.)
The blank lines should match.
