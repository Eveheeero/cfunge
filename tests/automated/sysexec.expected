Hi there:
Second command
Will run: exit 2

Retval (should be 0): 0 

Retval (should be 2): 2 
GOOD: = pushed -2 on zero length string.
